rector.php

```diff
@@ -4,6 +4,7 @@
 
 use Rector\CodeQuality\Rector\Class_\InlineConstructorDefaultToPropertyRector;
 use Rector\Config\RectorConfig;
+use Rector\Php74\Rector\Closure\ClosureToArrowFunctionRector;
 use Rector\Set\ValueObject\LevelSetList;
 
 return static function (RectorConfig $rectorConfig): void {
```

Comment:
I think it's better to start making our own rule set.
Just as an example: https://github.com/rectorphp/rector/blob/main/config/set/php80.php